eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml

```diff
@@ -1,18 +1,21 @@
+# cSpell:ignore changedfiles
+# cSpell:ignore Committish
+
 steps:
+  - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+    parameters:
+      DefaultBranchVariableName: DefaultBranch
   - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
     - pwsh: |
         Set-PsDebug -Trace 1
         # Find the default branch of the repo
-        $defaultBranch = (git remote show origin | Out-String) -replace "(?ms).*HEAD branch: (\w+).*", '$1'
+        Write-Host "Default Branch: $(DefaultBranch)"
         $targetBranch = "$(System.PullRequest.TargetBranch)" -replace "refs/heads/"
 
-        Write-Host "Default Branch: ${defaultBranch}"
-
-        if ((!"$(System.PullRequest.SourceBranch)".StartsWith("sync-eng/common")) -and $targetBranch -eq $defaultBranch)
+        if ((!"$(System.PullRequest.SourceBranch)".StartsWith("sync-eng/common")) -and $targetBranch -eq "$(DefaultBranch)")
         {
-
-          $filesInCommonDir = git diff "origin/${targetBranch}" HEAD --name-only -- 'eng/common/*'
-
+          $filesInCommonDir  = & "eng/common/scripts/get-changedfiles.ps1" -TargetCommittish $targetBranch -DiffPath 'eng/common/*'
+          # cSpell:ignore LASTEXITCODE
```

Comment:
Can these be put in one common place under eng/common or do they need to be next to the single line?

Reply:
Java is using `.vscode/cspell.json` for suppression.
I plan to use the same.

Reply:
Double check with logic. It is currently no ability to combine the setting file together with tool repo. Need to ignore on each file.